### PR TITLE
feat(users): add infoxicator

### PR DESCRIPTION
**Pull request by infoxicator**
## Proposed Changes

Add new user 

Look who's joined the party 😉

## Screenshots
Screenshot 2021-09-19 at 22 16 36

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/253"></a>

### CODE REVIEW

**Review by voidz7 — APPROVED** [review]
Looks good to me ⚡

File: public/data/infoxicator.json [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+      "icon": "github"
+    },
+    {
+      "name": "Follow me on Twitter",
+      "url": "http://twitter.com/infoxicador",
+      "icon": "twitter"
+    },
+    {
+      "name": "Learn more about Open Source on my YouTube channel",
+      "url": "http://youtube.com/rubencasas",
```
Comment (eddiejaoude) [inline public/data/infoxicator.json:18]: You have a YouTube channel 😱 , I did not realise, I have just subscribed 🎉

**Review by eddiejaoude — APPROVED** [review]
LGTM 👍

File: public/data/infoxicator.json (hunk 1 above)
Comment (infoxicator) [reply public/data/infoxicator.json:18]: I haven't posted in a while but I will get back to it at some point

**Pull request merged**